src/commands/broadcast-song/index.js

```diff
@@ -0,0 +1,72 @@
+const multiLocationSpecificSong = require("../multi-location-specific-song");
+const multiLocationSpotifySong = require("../multi-location-spotify-song");
+const multiLocationWildcardSong = require("../multi-location-wildcard-song");
+
+const displaySongChoice = async function(
+  bot,
+  updateMessage,
+  message,
+  { artist, name },
+  isDryRun,
+  locationName
+) {
+  const dryRunPrefix = isDryRun
+    ? "if this hadn't been a dry run :beach_with_umbrella: I'd have "
+    : "";
+
+  await updateMessage(
+    `Thanks <@${message.user}>, ` +
+      dryRunPrefix +
+      `queued "${name}" by ${artist} :man-with-bunny-ears-partying:,` +
+      ` in ${locationName}:round_pushpin:`
+  );
+};
+
+const init = function(listen, library, locations) {
+  /*
+   * Specific track search command, to queue song.  Queues the song in ALL locations.
+   * eg. @djroomba broadcast Under the Bridge by Red Hot Chili Peppers
+   * eg. @djroomba broadcast "Me and Julio down by the schoolyard" by "Paul Simon"
+   */
+  const broadcastSpecificSong =
+    '(dry run )?broadcast "?([^"]+)"? by "?([^"]+?)"?';
+  multiLocationSpecificSong(
+    listen,
+    library,
+    locations,
+    displaySongChoice,
+    broadcastSpecificSong,
+    "queue"
```

Comment:
❓ Should the broadcast command "play", rather than "queue"?

Reply:
There should probably be distinct commands for 'play everywhere' and 'queue everywhere'.  imho 'queue' is the far more useful of the pair, so that's the one I tied 'broadcast' to.  I can be convinced otherwise if it seems important.

Reply:
@RobKeep My expectation for a "broadcast" would be that it would play immediately. Not sure what the equivalent is for "queue everywhere" (or necessarily if that makes sense if queues differ on different sonos); if the intent is to keep a set of sonos queues in sync, perhaps a different tool would be better for that specific use case?